server/chatroom/chatroom_server/server_test.go

```diff
@@ -100,4 +196,64 @@ func TestGetChatroomsByUser(t *testing.T) {
 		assert(t, tt.want.Success, resp.Success)
 		assert(t, tt.want.Count, resp.Count)
 	}
-}
\ No newline at end of file
+}
+func TestLeaveChatroom(t *testing.T) {
+	prepareLeaveAndDeleteChatroomTestCtx()
+
+	s := server{}
+
+	ctx := prepareTestingCtx()
+
+	tests := []struct {
+		req  chatroompb.LeaveChatroomRequest
+		want chatroompb.LeaveChatroomResponse
+	}{
+		{
+			req: chatroompb.LeaveChatroomRequest{
+				Id: chatroomIdByToto,
+			},
+			want: chatroompb.LeaveChatroomResponse{
+				Success: true,
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		resp, err := s.LeaveChatroom(ctx, &tt.req)
+
+		check(err, "LeaveChatroom got unexpected error")
+
+		assert(t, tt.want.Success, resp.Success)
+
+	}
+
+}
+func TestDeleteChatroom(t *testing.T) {
+
+	s := server{}
+
+	ctx := prepareTestingCtx()
+
+	tests := []struct {
+		req  chatroompb.DeleteChatroomRequest
+		want chatroompb.DeleteChatroomResponse
+	}{
+		{
+			req: chatroompb.DeleteChatroomRequest{
+				Id: chatroomIdByAppliNH,
+			},
+			want: chatroompb.DeleteChatroomResponse{
+				Success: true,
+			},
+		},
+	}
```

Comment:
We should also test that a user who is not a chatroom owner cannot delete the chatroom.